backend/Watchdog.Core/Watchdog.Core.BLL/Models/ExampleTemplateData.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+
+namespace Watchdog.Core.BLL.Models
+{
+    // I used the basic sendgrid template, so the email is automatically sended in the promotions.
+    // To change template change it template id
+    public class ExampleTemplateData
+    {
+        [JsonProperty("subject")]
```

Comment:
You don't need to use JsonPropeperty if JSON object field has the same name as class files. It is case insensitive